scipy/stats/_binned_statistic.py

```diff
@@ -12,26 +12,28 @@
 
 
 BinnedStatisticResult = namedtuple('BinnedStatisticResult',
-                                   ('statistic', 'bin_edges', 'binnumber'))
+                                   ('statistic', 'bin_edges', 'binnumbers'))
```

Comment:
Sure, that makes sense.  I've changed these back.  I guess the general principle is that a change should be above some threshold of improvement to warrant a deprecation?